System/PathFinder.cpp

```diff
@@ -385,26 +368,17 @@ namespace RTE {
 		int firstY = std::floor((box.m_Corner.m_Y / static_cast<float>(m_NodeDimension)) + 0.5F) - 1;
 		int lastY = std::floor(((box.m_Corner.m_Y + box.m_Height) / static_cast<float>(m_NodeDimension)) + 0.5F) + 1;
 
-		// Truncate the influence
```

Comment:
Any clue what's up with the math for firstX and stuff. I.e. why the + 0.5F for flooring, then adding or subtracting 1?

I'm curious why Dan didn't just ceil instead of adding 0.5F then flooring? I guess that gets a different result, but it all seems pretty non-obvious to me haha

Reply:
Adding 0.5f and flooring is a round-to-closest, ceiling is different. Consider 0.1 -  that'd become 1 with a ceil, still 0.0 with a +0.5 and floor.

Anyways, kept this code the same as before 'cause it doesn't hurt and I don't want to spend more time debugging.

Reply:
oh derp, yeah, of course it is haha
I guess std::round is the nicer way of doing this, but I'm not suggesting you change it or anything, I was just curious what was up with it.